scanner/families/plugins/runner/runner.go

```diff
@@ -162,30 +163,30 @@ func (s *Scanner) isValidInputType(sourceType utils.SourceType) bool {
 	return false
 }
 
-func (s *Scanner) parseResults(ctx context.Context, runner types.PluginRunner) ([]apitypes.FindingInfo, error) {
+func (s *Scanner) parseResults(ctx context.Context, runner types.PluginRunner) ([]apitypes.FindingInfo, interface{}, error) {
 	result, err := runner.Result(ctx)
```

Comment:
can we return the actual result object instead only the raw data? we can access raw data from result where needed, but we also know the underlying types
```suggestion
func (s *Scanner) parseResults(ctx context.Context, runner types.PluginRunner) ([]apitypes.FindingInfo, *plugintypes.Result, error) {
	result, err := runner.Result(ctx)
```